src/components/JournalCard/style.css

```diff
@@ -0,0 +1,73 @@
+.journal-card {
+  padding: 13px;
+  background: rgba(255, 255, 255, 0.9);
+  box-shadow: 2px 3px 10px rgba(0, 0, 0, 0.1);
+  border-radius: 4px;
+  max-height: 140px;
+  font-family: 'VAGRounded', sans-serif;
+}
+
+.journal-card__top {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  font-weight: 500;
+  font-size: 13px;
+  color: #e9663a;
+  font-weight: 600;
+}
+
+.journal-card__date {
+  width: 40%;
+}
+
+.journal-card__time {
+  width: 30%;
+}
+
+.journal-card__delete {
+  width: 30%;
+  display: flex;
```

Comment:
what is the difference between line 21 and 29!

Reply:
the date contains text in it therefor it needs a bigger space on the card